heron/executor/src/python/heron_executor.py

```diff
@@ -1109,10 +1109,15 @@ def start_state_manager_watches(self):
     Receive updates to the packing plan from the statemgrs and update processes as needed.
     """
     Log.info("Start state manager watches")
+
+    with open(self.override_config_file, 'r') as stream:
+      overrides = yaml.load(stream)
+    overrides["heron.statemgr.connection.string"] = self.state_manager_connection
```

Comment:
Can `yaml.load` fail in any case? If so, should we check null and initialize `overrides` if necessary?

Reply:
You're right, if the file is empty, `yaml.load` returns `None`. Added a check, thanks!